src/features/navigation/RootNavigator/trustedDeviceRoutes.ts

```diff
@@ -25,11 +25,13 @@ export const trustedDeviceRoutes: GenericRoute<TrustedDeviceRootStackParamList>[
       path: 'securisation-compte/choix',
       parse: screenParamsParser['ReinitializePassword'],
     },
+    options: { title: 'Demande de suspension de compte' },
   },
   {
     name: 'SuspensionChoice',
     component: SuspensionChoice,
     path: 'securisation-compte/suspension',
+    options: { title: 'Demande de suspension de compte confirmation du choix' },
```

Comment:
Je trouve le wording bizarre et un peu long, proposition :
```suggestion
    options: { title: 'Confirmation de la demande de suspension de compte' },
```